pkg/dao/repository_configs.go

```diff
@@ -80,9 +83,17 @@ func (r repositoryConfigDaoImpl) Create(newRepoReq api.RepositoryRequest) (api.R
 
 	var created api.RepositoryResponse
 	ModelToApiFields(newRepoConfig, &created)
+
 	created.URL = newRepo.URL
 	created.Status = newRepo.Status
 
+	//Send update notification
```

Comment:
```suggestion
	//Send create notification
```
also okay with removing the comment because the code is named well enough to explain itself